tests/testthat/test-config-log.R

```diff
@@ -0,0 +1,94 @@
+context("Constructing config log from bbi_config.json")
+
+source("data/test-workflow-ref.R")
```

Comment:
We should change from the `source()` pattern to the `testthat` handling for special files, as described in metrumresearchgroup/mrgvalidatetestreference#8 (at least for all new development, if we don't want to change everything right now).